test/tests/pbs_reservations.py

```diff
@@ -0,0 +1,95 @@
+# coding utf-8
```

Comment:
I think you need a colon after "coding". TavisCI barfed on a non-ASCII character at line 28.